src/main/java/mServer/crawler/sender/arte/ArteJsonObjectToDatenFilmCallable.java

```diff
@@ -26,6 +26,9 @@
 {
     private static final Logger LOG = LogManager.getLogger(ArteJsonObjectToDatenFilmCallable.class);
     private static final String JSON_OBJECT_KEY_PROGRAM = "program";
+    private static final String JSON_ELEMENT_KEY_CATEGORY = "category";
+    private static final String JSON_ELEMENT_KEY_SUBCATEGORY = "subcategory";
+    private static final String JSON_ELEMENT_KEY_NAME = "name";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "JSON_ELEMENT_KEY_NAME" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)